README.md

```diff
@@ -204,16 +211,24 @@ Section 3.1 with the following exceptions:
   MUST publish on `Ga` a sequence of `group/add-members` messages according to
   [ssb-meta-feeds-group-spec] Section 3.1, such that the union of all recipients
   in that sequence equals all remaining members
-
-It is RECOMMENDED that epoch `G` is the "most preferred epoch" among all the
-epochs that `a` is a member of, which succeed a certain epoch zero.
+>   - TODO what is the `tangles.group` + `tangles.member` for these `group/add-member`
```

Comment:
I think if we pin down the tangles right, we can do really tidy quick lookups for the memberships of each epoch

Reply:
see this https://github.com/ssbc/ssb-group-exclusion-spec/issues/17